src/polymer/polymer-core-feature-scanner.ts

```diff
@@ -74,7 +74,7 @@ class PolymerCoreFeatureVisitor implements Visitor {
       feature.warnings.push(new Warning({
         message: `Expected assignment to \`Polymer.Base\` to be an object.` +
             `Got \`${rhs.type}\` instead.`,
-        severity: Severity.ERROR,
+        severity: Severity.WARNING,
```

Comment:
Seems like errors here are reasonable to me. They should only show up when something's really messed with with Polymer core?